modules/profile/templates/wordpress/docs/jquery-config.php.erb

```diff
@@ -14,3 +14,10 @@ define( 'JQUERY_STAGING_FORMAT', 'stage.%s' );
 if ( defined( 'XMLRPC_REQUEST' ) ) {
   @ini_set( 'memory_limit', '512M' );
 }
+
+<%- if @live_site == 'jquery.com' || @live_site == 'api.jquery.com' -%>
+define(
+  'JQUERY_BANNER',
+  'jQuery 4.0 is coming soon! Prepare by <a href="https://jquery.com/download/">upgrading</a> to the latest jQuery 3.x release. Learn more about our <a href="https://jquery.com/support/">version support</a>.'
```

Comment:
The best way to prepare would be to update to the latest jQuery 3.x, enable Migrate for a while and make sure no warnings are being issued.

One problem with this approach, though, is adding Migrate will also reintroduce jQuery 1.9 behavior in many APIs which is undesirable. I have an issue to tackle this problem in Migrate: https://github.com/jquery/jquery-migrate/issues/472.

For now, this is good, I think.